.github/pull_request_template.md

```diff
@@ -45,7 +45,7 @@ Fixes #
 2.
 3.
 
-* [ ] Do not include in the Testing Notes <!-- Check this box if this PR can't be tested by users (ie: it doesn't include user-facing changes or it can't be tested without manually modifying the code). -->
+* [ ] Should be tested by the development team exclusively <!-- Check this box if this PR should be tested by the development team exclusively (ie: it doesn't include user-facing changes or it can't be tested without manually modifying the code). -->
```

Comment:
Should we keep both boxes? @Aljullu made a good point about PRs not affecting the release, e.g. tests or build-related PRs

Reply:
Makes sense, updated 👍 !